### ISSUE TITLE
"全民瘋AI系列-經典機器學習"頁5-12，程式5.4將資料切分為訓練集與測試集

### ISSUE THREAD

**Issue by lancelotyang1225** [andy6804tw/crazyai-ml#9]
不好意思   請問一下  旨揭程式是否少了以下斜體+粗體字"分離特徵 X 和目標變數 y"的部分

from sklearn.model_selection import train_test_split

**_# 分離特徵 X 和目標變數 y
X = df_data.drop('target', axis=1)
y = df_data['target']_**

\# 將資料集分為訓練集和測試集
X_train, X_test, y_train, y_test = train_test_split(X, y, test_size = 0.3, random_state = 42)

print('Shape of training set X:', X_train.shape)
print('Shape of testing set X:', X_test.shape)

謝謝
